src/routes/participant/setlist/+page.ts

```diff
@@ -0,0 +1,62 @@
+import { db, verifyUserLoggedIn } from '$lib/firebase/client/firebase'
```

Comment:
jup, ik refactor het later wel weer gewoon lol

Reply:
Bedoel dit niet condescendingly trouwens, meer dat het gewoon weer dezelfde code is haha

Reply:
Veel plezier :p